eva-accession-release/src/test/java/uk/ac/ebi/eva/accession/release/io/MergedVariantMongoReaderTest.java

```diff
@@ -295,4 +297,44 @@ public void exceptionIfRsMergedHasNoSsMergeOperations() throws Exception {
                                                                        CHUNK_SIZE);
         readIntoMap(reader);
     }
+
+    /**
+     * This test will use a different defaultReader for assembly GCA_000004515.3 to evaluate this specific scenario:
+     * - Two merge operation for clustered variants
+     * - Two corresponding operations for its submitted variants
+     * - Only one of the clustered variants is active (Present in dbsnpClusteredVariantEntity collection)
+     *
+     * Only the clustered operation associated to the active clustered variant should be returned
+     */
+    @Test
+    public void excludeMergedIntoADeprecatedRs() throws Exception {
+        MergedVariantMongoReader reader = new MergedVariantMongoReader("GCA_000004515.3", mongoClient, TEST_DB,
+                                                                       CHUNK_SIZE);
+        Map<String, Variant> allVariants = readIntoMap(reader);
+        assertEquals(1, allVariants.size());
+        assertNotNull(allVariants.get("CM000851.2_2715437_G_A"));
+        assertNull(allVariants.get("CM000836.2_40018568_A_G"));
+    }
+
+    /**
+     * This test will use a different defaultReader for assembly GCA_000001111.1 to evaluate this specific scenario:
+     * - Two merge operations for clustered variants to the same RS IS (rs2222->rs1111 and rs3333->rs1111). Note that
+     * rs1111 is mapped to multiple locations (start:100 and start:200)
+     * - Two submitted variant operations indicating the merged operations of its RS ID
+     * - Only one clustered variant in the active collection (rs1111 start:100). This means that rs1111 start:200 has
+     * been deprecated
+     *
+     * Even though the rs1111 is only active with start 100, the merge reader will also allow the merged RS ID with
+     * start 200 in the merged VCF.
+     */
+    @Ignore
```

Comment:
can we put a message on why is this ignored? also, it was a bit hard for me to understand how is this test different to the previous one